### PR TITLE
Clean up dependencies and fix build

### PR DESCRIPTION
**Pull request by nacnudus**
* [x] There are too many dependencies.
* [x] Packrat isn't appropriate here.
* [x] The visualTest package is stale and should be replaced by `rlib/vdiffr`, which is maintained.

Anything else?

### CODE REVIEW

File: tests/testthat.R [hunk 1]
```diff
@@ -1,8 +1,6 @@
 library(testthat)
 library(govstyle)
 library(ggplot2)
-library(png)
-library(visualTest)
 # Install visualTest to allow checking against a reference plot.
```
Comment (matt-dray) [inline tests/testthat.R]: `library(visualTest)` has been removed – do we still need the 'Install visualTest to allow...' comment as well?

File: DESCRIPTION [hunk 2]
```diff
@@ -11,19 +11,18 @@ Description: Provides plotting funtions for reproducing plots in
...
+Suggests:
+    rmarkdown,
+    testthat,
+    dplyr,
+    readr,
+    knitr,
+    covr,
+    tidyr,
+    vdiffr
+Remotes: github::mangothecat/visualTest
```
Comment (matt-dray) [inline DESCRIPTION]: Does `visualTest` still need to be listed here in the description?

File: DESCRIPTION [hunk 3]
```diff
@@ -11,19 +11,18 @@ Description: Provides plotting funtions for reproducing plots in
...
-Remotes:  github::mangothecat/visualTest
+Suggests:
+    rmarkdown,
+    testthat,
+    dplyr,
+    readr,
+    knitr,
+    covr,
+    tidyr,
+    vdiffr
```
Comment (matt-dray) [inline DESCRIPTION]: Does this need shifting to remotes for now? I see that https://github.com/r-lib/vdiffr says in the readme

> Important: The CRAN version no longer works properly. Please use the development version which removes the dependency on the system FreeType library.

File: .gitignore [hunk 4]
```diff
@@ -1,10 +1,6 @@
+inst/doc
 .Rproj.user
 .Rhistory
 .RData
 .Ruserdata
-UD.zip
```
[the remark this answers is not in the document]
Comment (nacnudus) [reply .gitignore]: Good point, I'll add a clean-up operation to the vignette.

File: tests/figs/check-pal/base-graphics-histogram.svg [hunk 5]
```diff
@@ -0,0 +1,210 @@
+<?xml version='1.0' encoding='UTF-8' ?>
```
[the remark this answers is not in the document]
Comment (nacnudus) [reply tests/figs/check-pal/base-graphics-histogram.svg]: Yes.

File: README.Rmd [hunk 6]
```diff
@@ -8,7 +8,7 @@
 A package for applying a [gov.uk](http://govuk-elements.herokuapp.com/) style to plots created in the R package [ggplot2](https://github.com/hadley/ggplot2).
 This package *is in an early stage of development*; the intended end point is that it should be fully compliant with the gov.uk style guide.
 
-This package turns plots from this:
```
[the remark this answers is not in the document]
Comment (nacnudus) [reply README.Rmd]: Probably the mac vs linux newline convention.  I thought I'd configured Vim to ignore that, sorry.

File: DESCRIPTION (hunk 3 above)
Comment (nacnudus) [reply DESCRIPTION]: Good point.

**Pull request merged**